python-lib/gluonts_forecasts/training_session.py

```diff
@@ -30,7 +30,7 @@ class TrainingSession:
         timeseries_identifiers_names (list): Columns to identify multiple time series when data is in long format
         batch_size (int): Size of batch used by the GluonTS Trainer class
         num_batches_per_epoch (int): Number of batches per epoch
-        gpu (str): Not implemented
+        mxnet_context (mxnet.context.Context): MXNet context to use for Deep Learning models training.
```

Comment:
You need to import mxnet if you want to use type hints

Reply:
it's just a docstring not a type hint, so it won't fail